dotCMS/src/integration-test/java/com/dotmarketing/db/TomcatDatasourceStrategyTest.java

```diff
@@ -0,0 +1,33 @@
+package com.dotmarketing.db;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.zaxxer.hikari.HikariDataSource;
+import java.sql.SQLException;
+import javax.sql.DataSource;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/**
+ * Tests for {@link TomcatDataSourceStrategy}
+ * @author nollymar
+ */
+public class TomcatDatasourceStrategyTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5026767)